.changeset/plenty-monkeys-give.md

```diff
@@ -0,0 +1,16 @@
+---
+"@salt-ds/lab": minor
+---
+
+Added `DialogHeader` component to lab.
+
+```typescript
+  <Dialog open={open} onOpenChange={onOpenChange} id={id}>
+    <DialogHeader header="Terms and conditions" actions={<CloseButton />}/>
```

Comment:
Is this CloseButton existing core component or should be a generic button with close icon